spec/features/viewing_applications_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+include DateAndTimeMethods
+
+describe 'viewing job applications individually' do
+  context 'there are no pending applications or interviews' do
+    before :each do
+      when_current_user_is :staff, integration: true
+      visit staff_dashboard_url
+    end
+    it 'lets the user know there are no pending applications' do
+      expect page.has_text? 'No pending applications'
+    end
+    it 'lets the user know there are no pending interviews' do
+      expect page.has_text? 'No pending interviews'
+    end
+  end
+  context 'there are pending applications and interviews' do
+    let!(:unreviewed_record) { create :application_record, reviewed: false }
+    let!(:reviewed_record) { create :application_record, reviewed: true }
+    let!(:interview) do
+      create :interview,
+             application_record: reviewed_record,
+             scheduled: Time.zone.today
```

Comment:
Yes. I can be more specific if necessary, but believe me it does not go well to remove any of those let blocks.